frappe/website/doctype/web_form/templates/web_form.html

```diff
@@ -319,12 +327,12 @@ <h5 class='uppercase'>{{ _(section.label) }}</h5>
 					{{ _("Previous") }}</button>
 				{%- endif -%}
 				<!-- save/next button -->
-				{% if (loop.index == layout|len or frappe.form_dict.new) %}
```

Comment:
We should allow saving even if it is not new, update?

Reply:
This is correct. If the web form is new, the bottom button should be 'save' instead of the 'next'.